frontend/hub/useHubView.tsx

```diff
@@ -22,6 +22,11 @@ export interface HubItemsResponse<T extends object> {
   };
 }
 
+export interface HubNamespaceResponse<T extends object> {
```

Comment:
is this from an older version of another PR? This should be PulpItemsResponse, right?

Reply:
I'm using a branch from the shaiahwren-ns-list pr - should be the same on that one.